tests/unit_test/data/jobs/duplicate_clients/sag1/config/config_fed_client.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
just wondering if there is a way that we don't need to duplicate all these config_fed_client.json / config_fed_server.json

Reply:
always wondering if we should introduce pyhocon, which has include function